Client/Frontend/Home/HomePanelViewController.swift

```diff
@@ -16,6 +16,9 @@ private struct HomePanelViewControllerUX {
 
 protocol HomePanelViewControllerDelegate: class {
     func homePanelViewController(homePanelViewController: HomePanelViewController, didSelectURL url: NSURL, visitType: VisitType)
+    
+    //adds function to check panel changes in home screen
```

Comment:
You probably don't need a comment here either (`didSelectPanel` is pretty self-descriptive).